plugins.json

```diff
@@ -2,8 +2,157 @@
+            "default": null,
+            "hosting": ""
+          },
+          {
+            "key": "EncryptionKey",
+            "display_name": "At Rest Encryption Key:",
+            "type": "generated",
+            "help_text": "The AES encryption key used to encrypt stored access tokens.",
+            "placeholder": "",
+            "default": null,
+            "hosting": ""
+          },
+          {
+            "key": "GitlabGroup",
+            "display_name": "GitLab Group:",
+            "type": "text",
+            "help_text": "(Optional) Set to lock the plugin to a single GitLab group.",
+            "placeholder": "groupName",
+            "default": null,
+            "hosting": ""
+          },
+          {
+            "key": "EnablePrivateRepo",
+            "display_name": "Enable Private Repositories:",
+            "type": "bool",
+            "help_text": "(Optional) Allow the plugin to work with private repositories for subscriptions.",
+            "placeholder": "",
+            "default": null,
+            "hosting": ""
+          },
+          {
+            "key": "EnableCodePreview",
+            "display_name": "Enable Code Previews:",
+            "type": "dropdown",
+            "help_text": "Allow the plugin to expand permalinks to GitLab files with an actual preview of the linked file.",
+            "placeholder": "",
+            "default": "public",
+            "options": [
+              {
+                "display_name": "Enable for public projects",
+                "value": "public"
+              },
+              {
+                "display_name": "Enable for public and private projects. This might leak confidential code into public channels",
+                "value": "privateAndPublic"
+              },
+              {
+                "display_name": "Disable",
+                "value": "disable"
+              }
+            ],
+            "hosting": ""
+          }
+        ]
+      }
+    },
+    "platforms": {
+      "linux-amd64": {
+        "download_url": "https://plugins-store.test.mattermost.com/release/mattermost-plugin-gitlab-v1.8.1-linux-amd64.tar.gz",
+        "signature": "iQIzBAABCAAdFiEExViBuA9p6GO4WtXR0bVLR6XO/sQFAmXU70oACgkQ0bVLR6XO/sSrFQ/+J4EgmVgtCCDNmnSu3ep725qXLS+3GMmQKmAYrxFpxBTjXyhfdrw7HHR7QVk8U6zRIwP45zkxFHjoq6jC9KtfYT8CE62scpxc7jUhIercLdlwPrsAPocjHkCwy197egh3tWO/h1tEYCyHQ4N/YSmBKisJX/SgxRpQIQ1PmlWyRvvkN/7+VUzvsnVVzP+srrmuMoqXsR3YffBCFp3Rae4doLtrLPD7NEHI7FoDFqorad91N0h5wdSF81jp9aA3ClFCONBu4MODO/nU3zMXKJ6xhajMfXuSFYP1twopnjcRQAJOfA20ZYfqEO5Y5PX+nsep4ysYuMNhvcHLmu8lV26ZR1UsRbgp66DEFnuWrd+10dXw6huYouFML2gsFznatYI4NuwbOsZU65OuQMbQycac3y+qdZ7rr22zQzfJOLMCsM5WbNUh/oBLmE6www8kZJBZQuzqbg4o/8xKNXYOK00w8D0BKdrMmSCF+tDWh0wDOEfUREpDYgS1xsMcso4FlJzsopxuOdUREx+qcBINWli2K0Y3rmGu9y/lZzUytMiMz9SKlhkM6mXs5LL19ByIAOKFuykkzlb8KCP9hP1gX1Z5TYVow/Cn5jSdp9kDhskiw6Cmd7UNzvslr8cycl4bVAnPyx3cIfeA30+aIu0nItVKDUB0P6BSCcMjvPWCg/9sAIA="
+      },
+      "darwin-amd64": {},
```

Comment:
I think we can continue making marketplace releases with this issue, but we are blocked on prepackage PR CI runs with this right now. We will likely need to re-cut the releases, or skip these particular releases for prepackaging.